rules-tests/Php81/Rector/MethodCall/MyCLabsMethodCallToEnumConstRector/Fixture/usage_of_get_value.php.inc

```diff
@@ -8,7 +8,9 @@ final class UsageOfGetValue
 {
     public function run($value)
     {
-        $enum = SomeEnum::USED_TO_BE_CONST()->getValue();
+        $value = SomeEnum::USED_TO_BE_CONST()->getValue();
+        $enum = SomeEnum::USED_TO_BE_CONST();
+        $value2 = $enum->getValue();
```

Comment:
Please add another use can to be skipped when variable is not instance of came from `MyCLabs\Enum\Enum` instance

Reply:
New cases added